packages/@o3r/analytics/src/services/event-track/event-track.service.spec.ts

```diff
@@ -76,17 +74,6 @@ describe('Performance metrics', () => {
     return promise;
   });
 
-  // TODO to be fixed with the redesign of the event-track service
-  it.skip('should call different methods based on the NavigationEnd emissions', () => {
```

Comment:
Why removing it?
I know it's skip but it is here to be replaced (one day).
If a rule force you to remove it, we may disable the rule instead

Reply:
It was like that for 3 years and we discuss internally the fact to remove it .

Reply:
I am fine if this is not required by a linter rule

Reply:
As discussed we keep the rule, so devs must disable the rule locally with a description if they want to commit a skipped test